docs/book/v2/getting-started/skeleton.md

```diff
@@ -615,13 +616,17 @@ For the next part of the example, we'll be creating and wiring a factory for
 creating the `HelloAction` instance; the example assumes you used the default
 selection for a dependency injection container, zend-servicemanager.
 
-zend-servicemanager provides a tool for generating factories based on
-reflecting a class; we'll use that to generate our factory:
+zend-servicemanager provides a console (command line) tool for generating factories 
+based on reflecting a class; we'll use that to generate our factory. Navigate to your
+root skeleton-app directory in the console and enter the following:
 
 ```bash
-$ ./vendor/bin/generate-factory-for-class "App\\Action\\HelloAction" > src/App/Action/HelloActionFactory.php
+$ ./vendor/bin/generate-factory-for-class \
+> "App\\Action\\HelloAction" > ./src/App/src/Action/HelloActionFactory.php
 ```
 
+> For more information see [generate-factory-for-class](https://docs.zendframework.com/zend-servicemanager/console-tools/#generate-factory-for-class)
```

Comment:
Adding the link is good and imporant, but the additional blockquote is not needed. Please add the link to the paragraph above.